clang/test/CodeGenSYCL/int_header_ext_ascii.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2020 -fsycl-int-header=%t.h %s -o %t.out
+// RUN: FileCheck -input-file=%t.h %s
+
+#include "sycl.hpp"
+
+// Test ensures that we can properly handle unnamed kernels with extended-ascii
+// characters in it. Non ASCII characters aren't allowed in
```

Comment:
```suggestion
// characters in it. Non ASCII characters aren't allowed in.
```

Reply:
Think I'll just kill the incomplete sentence, no idea what i was going for.